store/conf/attrs/zimbra-attrs.xml

```diff
@@ -10104,4 +10104,9 @@ TODO: delete them permanently from here
     indexing         - Generate index
   </desc>
 </attr>
+
+<attr id="9030" name="zimbraCountObjectsEnabled" type="boolean" cardinality="single" optionalIn="globalConfig" since="11.0.0">
```

Comment:
Is there a possibility to disable counting objects except accounts (cos, domain etc.) in the future?
If the scope is Account object only, `zimbraCountAccountsEnabled` may be better.

Reply:
I do not see it can be disabled for cos, domain etc.
Will make change for variale name as zimbraCountAccountsEnabled

Reply:
Please rename it to `zimbraCountAccountsEnabled`.
If we need to add ability to disables other object, we will add a separate attribute.

Reply:
Done

Reply:
thanks.